### PR TITLE
Allow assigning to groups and fix some typos

### PR DESCRIPTION
**Pull request by mape2k**
Commit fixes issue #15 and some typos in german translation and github-URIs

### CODE REVIEW

**Comment by jperelli** [comment]
Hi mape2k, have you tested all this thoroughly?

File: app/models/periodictask.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 class Periodictask < ActiveRecord::Base
   unloadable
   belongs_to :project
+  belongs_to :assigned_to, :class_name => 'Principal', :foreign_key => 'assigned_to_id'
```
Comment (jperelli) [inline app/models/periodictask.rb]: Doesn't this change need a migration?

**Comment by mape2k** [comment]
I've tested this on redmine 2.5 - works flawlessly.

File: app/models/periodictask.rb (hunk 1 above)
Comment (mape2k) [inline app/models/periodictask.rb]: IMHO not, this does not change anything in database. It just links to class Principal (users-table) via foreign-key. As i did not found any database-migration examples for foreign-keys in redmine source i think, this does not need any migration. 

I did not migrate. Just clone master, apply patch and restart redmine.

Comment (jperelli) [inline app/models/periodictask.rb]: you are right

**Comment by jperelli** [comment]
Ok, I'm merging it. Thank you for all the fixes! :)

**Pull request merged**